ln/subscribe_invoice.ts

```diff
@@ -1,23 +1,27 @@
-const { subscribeToInvoice } = require('lightning');
-const { Order, User } = require('../models');
+import { Telegraf } from "telegraf";
+import { MainContext } from "../bot/start";
+import {subscribeToInvoice} from 'lightning'
+import { Order, User } from '../models';
 const { payToBuyer } = require('./pay_request');
-const lnd = require('./connect');
-const messages = require('../bot/messages');
+import { lnd } from "./connect";
+import * as messages from '../bot/messages';
 const ordersActions = require('../bot/ordersActions');
 const { getUserI18nContext, getEmojiRate, decimalRound } = require('../util');
-const logger = require('../logger');
+import logger from "../logger";
 
-const subscribeInvoice = async (bot, id, resub) => {
+const subscribeInvoice = async (bot: Telegraf<MainContext>, id: string, resub: boolean) => {
   try {
     const sub = subscribeToInvoice({ id, lnd });
     sub.on('invoice_updated', async invoice => {
       if (invoice.is_held && !resub) {
         const order = await Order.findOne({ hash: invoice.id });
+        if (order === null) return;
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program